docs/admin/releases/12/README.rst

```diff
@@ -0,0 +1,200 @@
+============================================
+ ESP Website Stable Release 12 release notes
+============================================
+
+.. contents:: :local:
+
+Changelog
+=========
+
+Privacy Policy
+~~~~~~~~~~~~~~
+The LU Board has approved a new `Privacy Policy <https://www.learningu.org/about/privacy/>`_, which governs how you collect and use the data on your site. These terms have been posted on your sites and must be followed.
+
+Attendance
+~~~~~~~~~~
+We've implemented a set of tools for teachers and admins to manage student class attendance:
+
+1. We've added a new page ``/teach/[one]/[two]/section_attendance`` that allows teachers to take attendance for a particular section (including unenrolled students). Teachers can either use a checkbox interface or scan barcodes (with their phones) like at student check-in to mark students as attending the class. Options are provided to allow teachers to enroll students in their class if they were not previously enrolled. The page is linked to from the main teacher registration page and from the webapp interface (for sections that are approved and scheduled). The page also has a dropdown menu for the teacher to select one of their sections for attendance. Attendance records for a class expire at midnight of the day they were set (this is useful for recurring classes, such that teachers can mark separate attendance for each instance of the same section). Splash at Yale made a `brief video tutorial <https://youtu.be/KV7i1G8s63k>`_ for this page.
+2. We've added an onsite module that summarizes attendance statistics/details. The page has a similar dropdown to select a particular timeslot for attendance. The page is linked to from the main onsite page (provided the module is enabled). The module will assume that you are interested in attendance records for the present day, but you can use the date picker to see a summary of attendance for a previous day (in the case of recurring classes). This page also has links to email, text, or get information for users that are checked in, playing hooky, or attending the wrong class.
+3. The onsite grid-based class changes will also show the number of students marked as attending a class (meaning there are now four numbers: "class attendance/program attendance/enrollment/capacity" for each class).
```

Comment:
how about "present in class/present at program/enrolled/capacity" to avoid confusion with overall program attendance?

Reply:
Or maybe "attending class/enrolled and attending program/enrolled/capacity"